doc/development/testing.rst

```diff
@@ -42,7 +42,7 @@ To run the test-suite, please proceed as follows.
    download the Python packages to be tested. For this please run::
 
      pip install -U -r tests/requirements-libraries.txt
-     py.test tests/unit tests/functional
+     pytest tests/unit tests/functional
 
 .. note:
```

Comment:
Can you move the docs changes to another PR please?

Reply:
@RS357 Please also rephrase the commit message first list. I read that you added a mentioning of deprecatde py.test. What this commit actually does is: "docs: Replace deprecated py.test by pytest".

Please try to describe hat the commit *does*, using some *verb* helps. This does not always hold, though, but is useful as a rule of thumb.